system/src/system_power_manager.cpp

```diff
@@ -330,6 +327,12 @@ void PowerManager::loop(void* arg) {
         // Do not re-run DPDM
         self->initDefault(false);
         self->update_ = true;
+      } else if (ev == Event::Wakeup) {
+        initDefault();
```

Comment:
Nit: might be a good idea to immediately acquire fuelgauge here rather than after `initDefault()`.